pymatgen/io/vasp/outputs.py

```diff
@@ -2750,8 +2750,7 @@ def p_ion(results, match):
                 self.er_bp_tot = self.er_bp[Spin.up] + self.er_bp[Spin.down]
 
         except Exception:
-            self.er_ev_tot = self.er_bp_tot = None
```

Comment:
Not sure we really need this?

Reply:
probably not. looks like it's set to that initially anyway

Reply:
Thanks for confirming!